TESTING.md

```diff
@@ -0,0 +1,121 @@
+# Testing with `nvim-treesitter`
+
+This document aims to provide some basic information on writing and running the `nvim-treesitter` test suite.
+
+## Setup
+
+### Install plenary and nvim-treesitter dependencies
+
+You'll need to have modules `plenary.nvim` and `nvim-treesitter` available on your runtime path.
+
+If you use packer, simply install plenary and nvim-treesitter normally.
+
+`vim-plug` users will need to add the `plenary.vim` file to the `plugin/` directory in addition to installing the module.
```

Comment:
`plenary.vim` file is a file of `plenary.nvim` itself
see https://github.com/nvim-lua/plenary.nvim/tree/master/plugin

Getting the file alone doesn't mean anything, the entirety of plenary.nvim is needed, and only that

Reply:
This is because `-u` makes it so it *skips* your `init.lua`, so it skips setting up `vim-plug`. What needs to be done here is to have `plenary.nvim` and `nvim-treesitter` in default `&rtp`, without needing an external tool like `vim-plug` to set in extras rtps

Or the alternative is to run `nvim -c "PlenaryBustedDirectory ...file_spec.lua"`, making sure that `plenary.nvim` and `nvim-treesitter` to be loaded when required

Reply:
Okay, I it looks like `minimal_init.lua` adds `nvim-treesitter` to the &rtp with this line

```lua
vim.opt.runtimepath:append "."
```

I'm having trouble figuring out how to add `plenary.nvim` to the &rtp without modifying `minimal_init.lua` and adding a line like

```lua
vim.opt.runtimepath:append "./plenary.nvim"
```

Do you have a standard way that you add `plenary.nvim` to the path?